data/dataDefaulter.js

```diff
@@ -42,7 +40,7 @@ function generateKey(entry) {
         if (entry[key]) {
             uniqueIdentifier += `${entry[key]
                 .toLowerCase()
-                .replace(/[^\w]/g, "")}|`;
+                .replace(/[^1-9bcdghj-np-tv-z]/g, "")}|`; // remove spaces, vowels, and 0
```

Comment:
I like where your head is at but I feel like this might be too lax and could cause bugs down the road...

I feel more strongly about not ignoring vowels than I do not ignoring 0s, but let me know how you feel.

The road/rd is a good call out but I'd personally prefer doing that on purpose, like road/rd, avenue/ave, street/st, circle/cir etc.

Reply:
Maybe we just match on less criteria for Walgreens specifically? Say `walgreenshyannis|{streetNum}|{zipWithoutZeros}` eg. `walgreenshyannis|52|261`. With this approach we do run the risk of having for instance a Walgreens with the same street number in the same zip code causing a collision.

Reply:
Or maybe this list helps. https://www.walgreens.com/storelistings/storesbycity.jsp?state=MA

Reply:
I was just trying to repair the persistent bug on the site.  I suppose we could also fix this on the front-end (if the data is never going away) by not showing appointments that are in the past.

Or, you can run this code ONE time and fix the file, then immediately revert to the old method.

Reply:
The first fix should cause the second Hyannis to be removed, since it doesn't have an associated timestamp yet. I'll merge this but revert this specific change, unless someone feels strongly about it or I'm missing something... (I'd normally wait for us to get consensus but I want to get this out tonight)